.github/workflows/trivy.yml

```diff
@@ -17,11 +17,15 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Build an image from Dockerfile
+      - name: Build Alpine image from Dockerfile
         run: |
-          docker build -t spacelift:${{ github.sha }} .
+          cd alpine && docker build -t spacelift:${{ github.sha }} .
```

Comment:
Can you not use the `-f <Dockerfile>` flag instead of `cd`? I'd normally try to avoid changing directory in a pipeline if possible.

Reply:
Sure I can switch to that, to be honest I decided to do it this way because I saw that's what was done in our `runner-pulumi` repo [here](https://github.com/spacelift-io/runner-pulumi/blob/main/.github/workflows/deploy-preproduction.yml#L44).